src/main/java/teammates/logic/core/EmailGenerator.java

```diff
@@ -87,6 +89,87 @@
     }
     
     /**
+     * Generates the summary of the feedback sessions (in which at least one
+     * email has been sent to students) email for the given {@code courseId} for {@code student}
+     */
+    public EmailWrapper generateFeedbackSessionResendAllLinksEmail(String courseId,
+            StudentAttributes student) {
+        
+        CourseAttributes course = coursesLogic.getCourse(courseId);
+        
+        List<FeedbackSessionAttributes> sessions = new ArrayList<FeedbackSessionAttributes>();
+        List<FeedbackSessionAttributes> fsInCourse = fsDb.getFeedbackSessionsForCourse(courseId);
```

Comment:
Do not communicate with `*Db` directly. Use `fsLogic` which is already defined for this purpose.